### PR TITLE
Update release process documentation [BT-91]

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: processes/release_processes/firecloud-develop.dot [hunk 1]
```diff
@@ -49,17 +49,22 @@ digraph {
 
   coordinate_release_timing [shape=oval label="If downtime will be required, co-ordinate with the release engineer\nand user support/comms about timings and banners"];
 
+  update_release_ticket [shape=oval label="Update the release ticket to add a link to the release notes"];
```
Comment (kshakir) [inline processes/release_processes/firecloud-develop.dot:52]: Historically, for some reason (a separate issue to be addressed elsewhere?) the GitHub release notes do NOT contain all of the Jira changes, thus may be insufficient for auditing purposes.

For the 55 release ticket, I had to run [generate my own diff](https://github.com/broadinstitute/cromwell/compare) to generate a fuller list of Jira tickets and issues that skipped auditing and weren't tracked in Jira.

**Review by kshakir — COMMENTED** [review]
TOL: I'm unsure why/how `processes/troubleshooting/all-purpose-mess-remover.dot.png` required re-rendering 🤷

**Review by grsterin — APPROVED** [review]

**Comment by mcovarr** [comment]
> TOL: I'm unsure why/how `processes/troubleshooting/all-purpose-mess-remover.dot.png` required re-rendering 🤷

Yeah me neither, perhaps I have a different version of `dot` than the previous committer...

**Comment by mcovarr** [comment]
@kshakir Updated the release doc with instruction for generating a commit list and adding the output to the release ticket.

**Review by kshakir — APPROVED** [review]

**Pull request merged**